libraries/v4_helper.rb

```diff
@@ -78,7 +79,7 @@ def feature_setup
         # Windows 10 v1703 (RS2)
         when '10.0.15063' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7]
         # Windows 10 & Server 2016 v1709 (RS3)
-        when '10.0.16299' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7 4.7.1]
+        when '10.0.16299' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7 4.7.1 4.7.2]
```

Comment:
Are you sure? it seems to me that 4.7.2 has been introduced as Windows Feature with 1803.
What is your source? :D

Also why do you keep package setup for 1709 (L128 to L132) if you add it to feature setup?
I really think only package setup should be kept